templates/components/matrix.build.hbs

```diff
@@ -0,0 +1,36 @@
+<div v-if="description" class="help-block description">\{{ description }}</div>
+<div class="fl-matrix">
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th v-for="(column, index) in columnOptions">\{{ column.label }}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(row, rowIndex) in rowOptions">
+        <th>\{{ row.label }}</th>
+        <td v-for="(column, colIndex) in columnOptions">
+          <div class="radio radio-icon" :class="{ 'readonly' : readonly }">
+            <input type="radio" class="matrix-radio-input" :id="getOptionId(rowIndex, colIndex, 'input')" :name="getOptionName(rowIndex)" :value="column.id" v-model.lazy="$v.value.$model" tabindex="-1">
+            <label v-on:click="readonly ? false : clickHandler(row, column, rowIndex, colIndex)">
+              <span class="check focus-outline"
+                :id="getOptionId(rowIndex, colIndex,'span')"
+                ref="matrix"
```

Comment:
According to https://vuejs.org/guide/essentials/template-refs.html#refs-inside-v-for this reference will be an array. I'm not sure what this is used for.

Reply:
followed template structure as of radio.build.hbs.

Reply:
@yogeshmsharma-xoriant This shows that your team follows existing code without a full understanding of how it works. This cannot happen.

Reply:
acknowledge,  Dev do understand the code they are writing but may lacking bit of UI expertise.